### ISSUE TITLE
Test and data

### ISSUE THREAD

**Issue by carlotita22** [med-air/3DSAM-adapter#18]
Hello! the github is very good, thank you very much.
I have the following doubts,

1. When I enter the data (colon), there is no data to test, so I don't know what the test command shows, what results can be seen with the test command? graphs? Segmentations?

2. To test your own data it is necessary to have the data pretrained? I don't have it, and the data I have is to segment the myocardium.

Regards!

**Comment by carlotita22** [comment]
and what type of information is the file data.pkl?

**Comment by peterant330** [comment]
> 1. ter the data (colon), ther

Hi,

1. We use the split.pkl to split the original training data into train/val/test, because the original Colon does not provide ground truth for test data. When you run the test command, it will show the dice/NSD metrics for the test data. You can also visualize the segmentation masks, which very simple modification of the code.
2. Yes, this is work aims to address the task of fast-finetuning, so you still need some training data for fine-tune.
3. I think we don't need data.pkl, we need a split.pkl to store the train/val/test split.

Best

**Comment by carlotita22** [comment]
Hi thanks for your answer! I have a other doubt, I want to try with my data but I have 4 channels for pacient because  i work with map t1, map 2 and respectivly contrast (heart). Can I use your github for this? What do I changes need?
